src/lib/app.js

```diff
@@ -56,8 +57,9 @@ class App {
     yield this.timerWorker.start()
     yield this.notificationWorker.start()
 
-    if (this.config.getIn(['db', 'sync'])) yield this.db.sync()
-    yield this.db.init()
+    if (this.config.getIn(['db', 'sync'])) {
```

Comment:
Notary uses environment variable process.env.NOTARY_RUN_MIGRATION to initiate migration.  
Perhaps we should change either Notary or Ledger to be consistent with how to initiate migration.